test/everlong/integration/CloseImmatureLongs.t.sol

```diff
@@ -156,11 +156,10 @@ contract TestCloseImmatureLongs is EverlongTest {
         // Close the long.
         uint256 baseProceeds = redeemStrategy(shares, bob, true);
 
-        assertGe(baseProceeds, estimatedProceeds);
-        assertApproxEqAbs(
+        assertApproxEqRel(
             baseProceeds,
             estimatedProceeds,
-            20,
```

Comment:
This should be the only real logic change in the PR...

After the updates elsewhere, these checks were failing in a negative interest rate environment...

With how the vaults work, we can't guarantee that `previewRedeem` always returns less than `redeem` since losses are applied during redemption.

Unsure of how this was passing before, but pretty sure it was due to being lucky with the fuzz cases.

Reply:
I don't see where the interest rate is set in this test. It looks like it's buried in `advanceTimeWithCheckpointsAndRebalancing`, but maybe it's set somewhere else. I think a version of this test that is just run in a positive interest environment would be useful to demonstrate that the tolerance is smaller in that case.

Reply:
Hey sorry for the late response on this... the setup functions must have gotten removed but not replaced.

When replaced (even with the original check) it passes so all is well :)

Reply:
Fixed in this pr: https://github.com/delvtech/everlong/pull/31

Reply:
Cool, everything else looks good. I'll approve here and check out the other PR.